### ISSUE TITLE
Project Page

### ISSUE THREAD

**Issue by Jonassala** [Verkami-Staff/TEST#7]
- Al compartir la página en redes, etc. Se utiliza el contenido de /manage_preoder - Título, imagen, etc. no el de /manage_settings.

**Comment by aitor** [comment]
> El título, subtítulo, imagen de portada y vídeo de la campaña no se pueden 'recuperar' en la vista como usuario. Ahora solo se muestran los de la preventa.

Esto no se muy bien como encaja con el pantallazo que habéis puesto en el que hay un enlace a "Preventa" y otro a "Sobre el proyecto".

**Comment by Jonassala** [comment]
Título, subtítulo, imagen de portada y vídeo son los subidos en el apartado de /manage_preorder siempre en la página de proyecto una vez está activa la misma. Pinches en el apartado que pinches. 

- Puedes recuperar la descripción del proyecto, pero no la portada, título y subtítulos originales de la campaña como usuario. 
- Como user podrías verlas en /manage_settings. 
- Se traspasan estos campos de /manage_settings a la página de /manage_preorder una vez se activa su visualización al conseguir el objetivo de la campaña, para que el autor las pueda modificar/ adaptar o mantener.

**Comment by sruiz7** [comment]
¿No sería mejor usar 'Sobre el proyecto' para poner ahí la info de preventa? Pensad que ahora en 'Sobre el proyecto' se describe el proyecto + las recompensas. Tendría sentido que al pasar a preventa el 'Sobre el proyecto' se describiera el proyecto + opciones de preventa. Lo comentamos ahora en la llamada.

**Comment by Jonassala** [comment]
# Diseños
**Active:**
![About (v1)](https://user-images.githubusercontent.com/2071003/80071680-53991380-8545-11ea-8d06-8e22d65ed026.png)
**Closed:**
![About (presale closed)](https://user-images.githubusercontent.com/2071003/80071705-5dbb1200-8545-11ea-8a24-4e2854a9b2a1.png)

- En el caso closed, faltaría bloquear las recompensas como hacemos ahora con las campañas terminadas. 
- Comentar el tema del código de colores de la barra debajo, si mantenemos el de las cards: 

![Captura de Pantalla 2020-04-23 a les 9 26 06](https://user-images.githubusercontent.com/2071003/80071900-b8546e00-8545-11ea-9ecd-9104adb916e3.png)

**Comment by Jonassala** [comment]
En relación con este diseño de la project page para preventas y la imagen de portada fullscreen, implementaríamos en Manage Preorder #3 de la siguiente manera: 

- Subes imagen de portada para desktop
- Se generan automáticamente las imágenes de: Responsive + Cards
- Una vez subida y generadas las imágenes, al refrescar la página se muestra en pequeñito el preview de responsive y de la card, generado automáticamente en base a la imagen de portada para desktop subida.
- Al lado de cada miniatura tienes la opción de: “Subir una imagen personalizada” (para Responsive y para Cards) para sustituir la que hemos generado de forma atomática.

> ! Falta todavía la solución definitiva para implementar en Manage Preoder pero irá por aquí.

Comentado más a fondo en Slack: https://linking.slack.com/archives/C029BBAMN/p1587725324028900

# Implementación de imagen de portada de proyecto fullscreen para campañas

En cuanto a esto también, nos gusta el diseño, y **nos gustaría implementarlo tanto para campañas como para preventas**.

Esto nos deja como en otras ocasiones en buscar una solución para las campañas antiguas.

Hemos pensado algunas posibilidades, pero seguro que puede haber muchas otras y podemos ver todas las que se nos ocurran para encontrar la mejor:

- Cambiar directamente las imágenes históricas a la nueva fullscreen (que tendrá las mismas dimensiones que la portada actual). Habrá ocasiones en que quedará raro, por tener mucho texto la imagen y quedar pisada con el texto del título y subtítulo. Pero son campañas terminadas.
- Mantener el diseño actual para las campañas antiguas
- Convertir las imágenes actuales añadiéndoles un marco negro, para que sigan con la misma dimensión, al estilo de: 

![Captura de Pantalla 2020-04-28 a les 17 52 26](https://user-images.githubusercontent.com/2071003/80509010-12f02e80-8979-11ea-94e1-820e5fde3cec.png)

Además de esto anteriormente comentado, se podría también:

- Dejar el front listo para aplicarlo a campañas (project page y manage_settings, pero en una primera fase, aplicar el fullscreen solo a preventas. 
- Una vez tengamos preventa en marcha, realizar el cambio para las campañas. Para que no nos encalle el tema de renderizar imágenes de portada, adaptarlas, etc. Y aplicar el cambio a las campañas como proyecto posterior, teniendo el diseño listo para implementar y ya funcionando en las preventas.

**Comment by aitor** [comment]
He estado revisando bien todo este asunto -haciendo transformaciones de prueba, una pequeña migración de unos cuantos proyectos de los comienzos de Verkami, ajustes en el upload- porque parece una parte fundamental del nuevo rediseño, la parte más "diferente" con respecto a lo que ya tenemos  y quería compartir algunas notas:

* Creo que si estamos de acuerdo en que a nivel de diseño es por dónde queremos ir lo que NO deberíamos hacer en ningún caso es mantener dos diseños. 

* De hecho creo que si vamos a tirar por esta via deberiamos _battle-testearlo_ en verkami.com con las campañas antes de deployar el tema de preventa.

* Para ello podríamos hacer es nueva versión en una plantilla (@sergiocerrutti eg`projects/new_hero`) y activarla sólo para algunos usuarios y/o proyectos de manera temporal. 

* En cuanto al tratamiento de la imagen:

  * Me sigue presentando dudas la usabilidad de los textos sobre la imagen no ya para el histórico sino en general para todos los autores actuales y pasados. Mi sensación es que les va a requerir de un trabajo de editorialización/dirección de arte/llámalo X que ahora no tienen.
  * Dicho esto si vamos a aplicar el fullscreen, yo tiraría por el reprocessing de las imagenes al tamaño que queramos, aceptando que algunas no quedarán bien. 
  * Aplicar el marco negro a la imagen no deberia ser necesario en ningun caso y si queremos usar esa opción visualmente podriamos hacerlo por CSS.

* Sobre el reprocessing çe principio -aunque llevaría unas cuantas horas con un proceso automático- no hay problema en reprocesar las portadas de todos los proyectos.

@sergiocerrutti @sruiz7:

* ¿Me podéis dar acceso a los sketchs? Me gustaria poner aqui un mockup de como quedan algunos proyectos emblemáticos históricos en el nuevo diseño.
* ¿Tenemos ya unos tamaños de imagen decididos para ese nuevo fullscreen?

**Comment by sruiz7** [comment]
> @sergiocerrutti @sruiz7:
> 
> * ¿Me podéis dar acceso a los sketchs? Me gustaria poner aqui un mockup de como quedan algunos proyectos emblemáticos históricos en el nuevo diseño.
> * ¿Tenemos ya unos tamaños de imagen decididos para ese nuevo fullscreen?

Yes, te dejo el link al sketch: https://www.dropbox.com/s/73n6hr8l9eai7vo/Preventa.sketch?dl=0

Sobre lo que comentas de las imágenes, están al 80% de opacidad sobre el fondo negro para asegurar legibilidad. Lo que sí habrá que "guiar" a los creadores es a que no usen fotos/diseños con muchos textos, que esto sea más a modo "portada de libro".

En cuanto al tamaño, mantenemos la misma proporción de 16:9 y utilizamos un background-size cover.

**Comment by aitor** [comment]
> Sobre lo que comentas de las imágenes, están al 80% de opacidad sobre el fondo negro para asegurar legibilidad. 

Umm… ok me habia parecido que no habia ningun procesamiento en las imágenes de ejemplo. Usaré eso en el mockup y vemos ejemplos reales.

> En cuanto al tamaño, mantenemos la misma proporción de 16:9 y utilizamos un background-size cover.

Si bueno, me refería al tamaño exacto que vamos a usar. Entiendo que el tamaño que estamos generando ahora de `940x530` no va a dar la calidad mínima necesario para ir a _edge-to-edge_ en escritorio y tenemos que irnos al menos a `1280 x 720` o incluso `1600 x 900`. 

En cuanto antes me podeis pasar estas especificaciones antes podemos preparar la migración histórica y ver casos reales.
